src/setup/index.js

```diff
@@ -35,6 +37,13 @@ export default function () {
         },
     });
 
+    // force app layout to work left to right.
+    I18nManager.allowRTL(false);
+    I18nManager.forceRTL(false);
+    I18nManager.swapLeftAndRightInRTL(false);
+    TextInput.defaultProps = TextInput.defaultProps || {};
+    TextInput.defaultProps.textAlign = 'left';
```

Comment:
Interesting Hack... I am fine with it. @marcaaron Do you think we can do this?

Reply:
if you think about it android does something like this from native. when applying force RTL mode.

Reply:
If it's the `defaultProps` stuff. I'm not really sure I get why we overwriting `defaultProps` with itself... 

In which cases would `TextInput` as imported directly from `react-native` not have any `defaultProps`?

If it depends on platform then we are doing this wrong and should apply the default prop in our specific implementation for the `TextInput` and not add this here.

Reply:
At the very least I'd expect a comment here to explain why it's necessary @parasharrajat

Reply:
@marcaaron  

I have had this issue before and that solves it for me it's generally on react native not related to the platform  . so I add it to avoid this issue.  

https://stackoverflow.com/questions/54909798/undefined-is-not-an-object-evaluating-reactnative-text-defaultprops-allowfron/55634893

Reply:
Yeah. Sorry, missed to request the comments. Due to many iterations of solution on this PR, It got slipped.

Reply:
What is the summary of the problem? Is it that some platforms do not have `defaultProps`? Can we please fix it by using the platform extensions mentioned here?

https://github.com/expensify/app#platform-specific-file-extensions

Reply:
no, it's not related to the platform. 

![Screen Shot 2022-02-23 at 3 20 09 PM](https://user-images.githubusercontent.com/21364901/155327348-01f1c13b-8fa9-4e0b-9217-f92f4a764862.png)

Reply:
@ahmdshrif Could you please give a brief summary here?

Reply:
Eh I still disagree with the placement of this code. Why is it added in our `setup()` method? We use an abstraction around `TextInput` and do not really use the `TextInput` directly from RN see here:

https://github.com/Expensify/App/blob/87230549a3d9c299f7abbc98012f4143383c2c19/src/components/TextInput/BaseTextInput.js#L220-L246

Can we just force `textAlign="left"` instead of using this `defaultProps` hack?

Reply:
ok but that is mean we need to do this on all of our text input components since all of them implement from react naive directly not from this file. 

like this 

https://github.com/Expensify/App/blob/2f14cb7fad40f00869b844ed3e31693a48318018/src/components/TextInputFocusable/index.android.js#L3
and we could have a new component.

Reply:
I agree. Let's do that @ahmdshrif. There is a reason for doing that. Setup is not really a suitable place to put JSX related code. (that's why I called it to hack).  

Update all Native TextInput references in the code base pass the prop directly as suggested and add a comment to this :arrow_down: a place to explain why we added this prop. I am sure others will find that helpful.

https://github.com/Expensify/App/blob/87230549a3d9c299f7abbc98012f4143383c2c19/src/components/TextInput/BaseTextInput.js#L220-L246

Reply:
>that is mean we need to do this on all of our text input components since all of them implement from react naive directly not from this file

Yep, sounds like you understood the task now, thank you!